views/layout.pug

```diff
@@ -44,6 +40,10 @@ html(lang="en")
 							span.c-navbar-menu-text Home
 		block content
 		block foot
+			script(src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js")
+			script(src="https://unpkg.com/vue@2.0.3/dist/vue.js")
+			script(src="https://cdn.jsdelivr.net/g/vue@2.0.0-rc.4,vue.resource@1.0.0")
+			script(src='/js/script.js')
 			footer.c-footer
```

Comment:
you mean footer div needs to be below the footer block and inside the div I should have all the scripts?